src/routes/RootRouter/RootRouter.js

```diff
@@ -57,6 +56,7 @@ export default function RootRouter () {
   }
 }
 
+// Move into `PublicLayoutRouter`
```

Comment:
The delineation between `NonAuthRouter` and `PublicRouter` I do think is important. `NonAuthRouter` has a lot of special cases related to Sign Up and Auth in general. It's for all the things we do in those worlds. `PublicRouter` I expect will continue to grow and take a different shape increasingly similar to `AuthLayoutRouter`... I'm hoping we can keep `RootRouter` telling this top-level story well, I think it'll serve us. Delegate Public routes and routing business to `PublicLayoutRouter`. I can help show how to do that well in this case I think.